src/applications/disability-benefits/all-claims/components/ArrayField.jsx

```diff
@@ -0,0 +1,427 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash/fp';
+import classNames from 'classnames';
+import Scroll from 'react-scroll';
+
+import {
+  toIdSchema,
+  getDefaultFormState,
+  deepEquals,
+} from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
+
+import { scrollToFirstError } from '../shared/utilities/ui';
+import { setArrayRecordTouched } from '../shared/helpers';
+import { errorSchemaIsValid } from '../shared/validation';
```

Comment:
Rather than copying these files to vets-website, what do you think about importing them from USFS?